src/common/solution-info.ts

```diff
@@ -27,9 +27,22 @@ export class SolutionInfo {
   static SOLUTION_VERSION_SHORT = parseVersion(SolutionInfo.SOLUTION_VERSION).short;
   static DESCRIPTION = `(${SolutionInfo.SOLUTION_ID}) ${SolutionInfo.SOLUTION_NAME} ${SolutionInfo.SOLUTION_VERSION_DETAIL}`;
   static SOLUTION_TYPE = 'AWS-Solutions';
+
+  static V1_0_0 = 'v1.0.0';
```

Comment:
manage versions like this,

https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_rds.PostgresEngineVersion.html